pyqg/kernel.pyx

```diff
@@ -295,7 +295,7 @@ cdef class PseudoSpectralKernel:
         self.ifft_qh_to_q()
         # input might have been destroyed, have to re-assign
         self.qh[:] = b_view
-        
+
```

Comment:
The fact that you are mixing whitespace cleanup with major changes to the core computational routines makes the code quite hard to review.